src/ServiceControlInstaller.Engine/Services/WindowsServiceController.cs

```diff
@@ -35,7 +35,7 @@ public string Account
                 var account = (string)ReadValue("ObjectName");
                 if (account == null)
                 {
-                    throw new NullReferenceException("ServiceAccount entry not found in the registry");
+                    return null;
```

Comment:
If account is null and we want to return null, you don't need the null check at all - just return account